### DISCUSSION TITLE
Pages in subfolders?

### DISCUSSION THREAD

**Discussion by nanawel** [sqlpage/SQLPage#18] [General]
I'm trying to break my pages down logically and I thought that putting them in subfolders in order to access them with `mypath/mypage.sql` would be okay.

Well, it is, but the style is missing (at least with the Docker image) because the page tries to access the CSS and JS bundles in the `mypath/` folder, while those files are only in the root folder.

Is it a bug or a configuration I missed somewhere?

**Comment by lovasoa** [comment]
That is an issue with SQLPage. I'll fix it in the next version: https://github.com/lovasoa/SQLpage/issues/19
